content/dart/concepts/type-conversion/terms/toStringAsFixed/toStringAsFixed.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.toStringAsFixed()'
+Description: 'Converts a numeric value into a string representation'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Mobile Development'
+Tags:
+  - 'Dart'
+  - 'Data Types'
+  - 'Methods'
+  - 'Numbers'
+  - 'Strings'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, **`.toStringAsFixed()`** is a method used to convert numeric [data types](https://www.codecademy.com/resources/docs/dart/data-types), typically used with `double`, to a string representation with a fixed number of decimal places. If the number is an integer it will be converted to a `double` before computing the string representation.
```

Comment:
```suggestion
In Dart, **`.toStringAsFixed()`** is a method used to convert numeric [data types](https://www.codecademy.com/resources/docs/dart/data-types), typically used with `double`, to a string representation with a fixed number of decimal places. If the input number is an integer, it will first be converted to a `double` before computing the string representation.
```